### PR TITLE
add list command & other minor changes

### PR DESCRIPTION
**Pull request by houqp**
* list command to display all dockerfiles managed by floydker
* add flag to build command to generate image tag from dockerfile path
* remove build container after build/test is done

### CODE REVIEW

File: floydker/src/floydker/list_cmd.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+@click.argument('search_root')
+@click.option('--tag-only/--no-tag-only',
+              help='only display image tags',
+              default=False)
+@click_log.simple_verbosity_option()
+@click_log.init(__name__)
+def tag(search_root, tag_only):
+    for directory in find_project_dirs(search_root):
+        if not tag_only:
+            print(directory + ':')
```
Comment (narenst) [inline floydker/src/floydker/list_cmd.py]: Just wanted to check. Should this exit out when tag_only is true?

File: floydker/src/floydker/list_cmd.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+@click.argument('search_root')
+@click.option('--file-only/--no-file-only',
+              help='only display dockerfile names',
+              default=False)
+@click_log.simple_verbosity_option()
+@click_log.init(__name__)
+def dockerfile(search_root, file_only):
+    for directory in find_project_dirs(search_root):
+        if not file_only:
+            print(directory + ':')
```
Comment (narenst) [inline floydker/src/floydker/list_cmd.py]: And should this also exit when "file_only" is True?

**Comment by narenst** [comment]
LGTM!

**Comment by houqp** [comment]
Yes, I have updated the comment to document the behaviors of those options ;)

**Pull request merged**